src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -35,296 +35,650 @@
 */
 public class UTIL_ACCT_Naming {
     /*******************************************************************************************************
-    * @description The user defined Administrative Account record type.
+    * @description The default Account type.
     */
-    private static ID userDefinedAdminRecordTypeId = Util_Describe.getAdminAccRecTypeID();
+    private static ID defaultRecTypeId = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c;
 
     /*******************************************************************************************************
-    * @description update admin account names if any changes made to contacts' lastname
-    * @param listContactsChangedLastName list of contacts whose lastnames are changed
-    * @return list<Account>
+    * @description The Household Account record type.
     */
-    public static list<Account> updateAdminAccountsName(List<Contact> listContactsChangedLastName) {
-        List<Account> listAccToUpdate = new List<Account>();
-        Map<Id, Contact> mapAccIdToCon = new Map<Id, Contact>();
-        for (Contact con : listContactsChangedLastName) {
-            mapAccIdToCon.put(con.AccountId, con);
+    private static ID hhAccountRecordTypeId = UTIL_Describe.getHhAccRecTypeID();
+
+    /*******************************************************************************************************
+    * @description The Administrative Account record type.
+    */
+    private static ID adminAccountRecordTypeId = UTIL_Describe.getAdminAccRecTypeID();
+
+    /*******************************************************************************************************
+    * @description The default/selected Household Account name format.
+    */
+    private static String hhNameFormat = UTIL_CustomSettingsFacade.getSettings().Household_Account_Naming_Format__c;
+
+    /*******************************************************************************************************
+    * @description The default/selected Administrative Account name format.
+    */
+    private static String aaNameFormat = UTIL_CustomSettingsFacade.getSettings().Admin_Account_Naming_Format__c;
```

Comment:
thank you for putting these into variables :) I hate reading the whole call to UTIL as part of if conditions etc and would rather read the variable.